### DISCUSSION TITLE
Clarify Store Pets/Mounts/Equipment on bind status in the item title, description, or checkout process

### DISCUSSION THREAD

**Discussion by andovcr** [chromiecraft/chromiecraft#5876] [Suggestions]
Could someone go through and tag each pet/mount/equippable item on the shop with BoE, BoP, BoA, etc. so that players can understand the limitations of the item before purchasing it?

Long-term, I would love if staff would review their earlier stance on [Make chromie store Mounts/Pets account wide](https://github.com/chromiecraft/chromiecraft/discussions/3234), but in the meantime, I think this is a fair request.

If manpower is the issue, then I would be HAPPY to receive unlimited CC points and doing the testing for you :P

**Comment by Vyperhand** [comment]
Slight expansion - the store needs more detail across the board - services should lay out exactly what you do and don't get, as well.

**Comment by Amrx2** [comment]
Hi both, mounts already specify their bind status in the description for all / which faction it is available for etc.

There was a specific issue with the Corrupted Kitten being a common item, it should have been sent bind on pickup as with all the other pets (minus a few which are BoA, I do need to update that for them). All items are sent bind on pickup unless it is bind on account, specifically so that people cannot buy shop items then sell it on the AH for gold. Early on people were buying mounts and bags and selling it on the AH, thus we changed all items when sent specifically via the shop to be  BoP / BoA. 

@Vyperhand that is on my list of things to do still, sorry I will eventually get around to it (did mention this on Discord but making everyone else aware it is on the to-do list. Will see if we can do something on the PTR in terms of commands to trial stuff from the shop (like we did with mogs with the NPC) but is a lower priority than most items that currently.

**Reply by Vyperhand** [reply]
Thanks.  I got a surprise on a race change recently, brought it top of mind.  If I had details on how it would have worked, I would still have bought it - just 3 levels later.  :_(  

Appreciate that it's on the list, thanks.
